.env.local

```
@@ -0,0 +1,3 @@
+LAMATIC_API_KEY=lt-9c7eb72fb10a91d48a7b0312b4fe9d65
+LAMATIC_ENDPOINT_URL=https://aroojsorganization327-aroojsproject386.lamatic.dev/graphql
+LAMATIC_WORKFLOW_ID=163efbc7-595e-4b3a-8b2a-ae8bf3c2a8b0
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)